### PR TITLE
fix(docs): Angular API & integration Doc page TG-445

### PR DESCRIPTION
**Pull request by JanCizmar**

### CODE REVIEW

File: docs/integrations.mdx [hunk 1]
```diff
@@ -33,6 +33,18 @@ Tolgee has currently implemented these integration libraries.
 - [GitHub Repository](https://github.com/tolgee/tolgee-js/tree/main/packages/svelte)
 - [Example App](https://github.com/tolgee/svelte-example)
 
+## Next
+
+- [Docs](/docs/web/using_with_react/ssr/using_with_next)
+- [Langing Page](/integrations/next)
```
Comment (marketachalupnikova) [inline docs/integrations.mdx:39]: landing

File: docs/integrations.mdx [hunk 2]
```diff
@@ -33,6 +33,18 @@ Tolgee has currently implemented these integration libraries.
...
+- [Example App](https://github.com/tolgee/next-example)
+
+## Gatsby
+
+- [Docs](/docs/web/using_with_react/ssr/using_with_gatsby)
+- [Langing Page](/integrations/gatsby)
```
Comment (marketachalupnikova) [inline docs/integrations.mdx:45]: landing

File: docs/integrations.mdx [hunk 3]
```diff
@@ -33,6 +33,18 @@ Tolgee has currently implemented these integration libraries.
...
+- [Example App](https://github.com/tolgee/gatsby-example)
+
 ## JS Core
 `@tolgee/core` npm packages is used by all JavaScript integration, so if you're building your custom
```
Comment (stepan662) [inline docs/integrations.mdx:49]: npm **package** is used ...

**Review by stepan662 — APPROVED** [review]

File: docs/web/using_with_angular/api.mdx [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+`EventEmitter<never>` emitting every time language is changed.
+
+### onTranslationChange
+`EventEmmitter<TranslationData>` emitting when translation is changed.
+
+#### type `TranslationData`
+```ts
+{
+    key: string; // the key which was chenged
+    translations: { // new translatois for language od languages
```
Comment (marketachalupnikova) [inline docs/web/using_with_angular/api.mdx:130]: translations, or

**Pull request merged**